api/src/test/java/org/pmiops/workbench/mail/MailServiceImplTest.java

```diff
@@ -0,0 +1,117 @@
+package org.pmiops.workbench.mail;
+
+import com.google.api.services.admin.directory.model.User;
+import com.google.api.services.admin.directory.model.UserEmail;
+import com.google.api.services.admin.directory.model.UserName;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.Mock;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.google.CloudStorageServiceImpl;
+import org.pmiops.workbench.google.CloudStorageService;
+import org.pmiops.workbench.mandrill.api.MandrillApi;
+import org.pmiops.workbench.mandrill.ApiException;
+import org.pmiops.workbench.mandrill.model.MandrillApiKeyAndMessage;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatuses;
+import org.pmiops.workbench.mandrill.model.MandrillMessageStatus;
+import org.pmiops.workbench.test.Providers;
+
+import javax.inject.Provider;
+import javax.mail.MessagingException;
+import javax.validation.constraints.NotNull;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class MailServiceImplTest {
+
+  private MailServiceImpl service;
+  private static final String GIVEN_NAME = "Bob";
+  private static final String FAMILY_NAME = "Bobberson";
+  private static final String CONTACT_EMAIL = "bob@example.com";
+  private static final String PASSWORD = "secretpassword";
+  private static final String PRIMARY_EMAIL = "bob@researchallofus.org";
+  private static final String API_KEY = "this-is-an-api-key";
+
+
+  @Mock
+  private CloudStorageServiceImpl cloudStorageService;
+
```

Comment:
Minor nit: The convention in our tests is to have no line breaks between the mocks.